src/cryptoadvance/specter/templates/includes/tx-table.html

```diff
@@ -1018,8 +1018,8 @@ <h1>{{ _("Export transactions to CSV") }}</h1>
                         let frozenImgClone = txRow.frozenImg.cloneNode()
                         txRow.callbackaction.append(frozenImgClone)
                         // Adding "Frozen" in coin selection and UTXO overview
-                        txRow.callbackaction.append("Frozen")
-                        txRow.categoryCell.append("Frozen")
+                        // txRow.callbackaction.append("Frozen")
```

Comment:
If we remove it, do we need some hover over / tooltip to explain the icon?

Reply:
No, i think it's clear enough with the icon

Reply:
OK